File: codes/if-else.c
#include <stdio.h>

int main() {
    int num;
    num = 6;

    if (num >= 0) {
        printf("El número es positivo.\n");
    } else {
        printf("El número es negativo.\n");
    }

    //opcional
    if(num >= 0 && num <= 9) {
		printf("El valor tiene un digito.\n");
	} else {
		printf("El valor tiene mas de un digito.\n");
	}

    return 0;
}

File: output/if-else.py
def main():
	
	num = 6
	if(num>0):
		print("El número es positivo.\n")
		
	elif(num==0):
		print("El número es cero.\n")
		
	else:
		print("El número es negativo.\n")
		
	if(num>=0 and num<=9):
		print("El valor tiene un digito.\n")
		
	else:
		print("El valor tiene mas de un digito.\n")
		
	return 0
	
if __name__ == '__main__':
	main()
